generators/client/templates/angular/package.json.ejs

```diff
@@ -154,7 +154,7 @@
     <%_ } _%>
     <%_ if (cypressTests) { _%>
     "e2e:cypress:headless": "<%= clientPackageManager %> run e2e:cypress -- --headless",
-    "e2e:cypress": "cypress run --browser chrome",
+    "e2e:cypress": "cypress run --browser chrome --record ${CYPRESS_ENABLE_RECORD:-false}",
```

Comment:
Is this portable?
Scripts starting with `ci:` are not portable.
But for this script IMO it should be.

Reply:
good question, I don't know. But I did not succeed on use arguments with the current "ci:" scripts

Reply:
See original bug about this https://github.com/jhipster/generator-jhipster/issues/12805.
Have you tried `concurrently -k -s first "npm run ci:e2e:server:start" "npm run e2e:headless -- --record ${CYPRESS_ENABLE_RECORD:-false}"`

Reply:
I have tested it but it doesn't work (If I remember I have already tested this before).

You can check logs here:
https://github.com/avdev4j/jhipster-cypress-dashboard/runs/1810120628?check_suite_focus=true

![Capture d’écran 2021-02-01 à 22 50 58](https://user-images.githubusercontent.com/37835668/106522475-17d1a800-64e0-11eb-8f78-075819a36955.png)